src/rules/block-no-single-line/README.md

```diff
@@ -0,0 +1,56 @@
+# block-no-single-line
+
+Disallow single-line blocks.
+
+```css
+  a { color: pink; top: 0; }
+/** ↑                      ↑
+ * Declaration blocks like this */
+```
+
+The following patterns are considered warnings:
+
+```css
+a { color: pink; }
+```
+
+```css
+a,
+b { color: pink; }
+```
+
+```css
+a { color: pink; top: 1px; }
+```
+
+```css
+@media print { a { color: pink; } }
+```
+
+```css
+@media print {
+ a { color: pink; }
+}
+```
```

Comment:
The rule will flag up the single line nested at-rule and not the `a` rule. I've added tests and examples to the readme.